spec/plans/provision.fmf

```diff
@@ -137,6 +163,8 @@ example: |
             hostname or ip address of the box
         user
             user name to be used to log in, ``root`` by default
+        become
+            whether to run scripts with ``sudo``, ``false`` by default
```

Comment:
Do we want to mention that 'sudo' is not used if the user is 'root'?

Reply:
I pushed an update specifying it. However, seeing @happz  comment about the coming PR, I wonder if I should instead put that note everywhere now in this PR as well.

Reply:
@lukaszachy can you pls re-review?